services/web/src/modals/EditProduct.js

```diff
@@ -160,12 +161,12 @@ export default class EditProduct extends React.Component {
                 onError={(error) => this.setState({ error })}
               />
               {!this.props.shop && (
-                <ReferenceField
+                <SearchDropdown
                   required
                   name="shop"
                   label="Shop"
                   value={product.shop}
-                  path="/1/shops/search"
+                  searchPath="/1/shops/search"
```

Comment:
is `searchPath` more meaningful than just `path` inside the "search" dropdown?

Reply:
I this case i was following the Semantic UI naming e.g. 
https://react.semantic-ui.com/modules/dropdown/

onSearchChange
searchInput
searchQuery

Reply:
kk makes sense